sdk/eventhub/.azure-pipelines/pr.client.yml

```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT THIS FILE
```

Comment:
Do you think it is worth scoping these under a ".azure-pipelines" folder? I'm not necessary against it but I'm curious if there is a reason you had in mind. On potential issue I can see is that we might need to add some exceptions to some of or tools as they are going to be assuming each directory under the service folder is a shipping package.

Reply:
I think we can take two approaches, one is to have a structure like the following:

```
.azure-pipelines/
  components/
    servicebus/
      pr.client.yml
  templates/
    jobs/
      archetype-sdk-client.yml
sdk/
  servicebus/
    (actual client code)
```

or something like this (current PR):

```
.azure-pipelines/
  templates/
    jobs/
      archetype-sdk-client.yml
sdk/
  servicebus/
    .azure-pipelines/
      pr.client.yml
    (actual client code)
```

One of the pros for the first approach is that it puts all the pipeline related configuration under one folder, and probably discourages changes by devs. I've gone with the second approach for now but happy to flick it back. I don't have a strong preference at this point.

Reply:
I like having all the things related to a given service under the service directory that wasn't my question. I was more wondering why you did it under `sdk/servicebus/.azure-pipelines/pr.client.yml` vs `sdk/servicebus/pr.client.yml`

Reply:
Oh - that was just in case we need to have seperate files in there (just in case).

Reply:
After some thought I am going to remove the .azure-pipelines folders under each service directory. We can add it back in if we ever have the need but its not necessary right now since in most cases the files themselves should be all that is needed.